# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======

ScoopLitterboxActionGoal action_goal
ScoopLitterboxActionResult action_result
ScoopLitterboxActionFeedback action_feedback
